tox.ini

```diff
@@ -1,9 +1,9 @@
 [tox]
-envlist = flake8,pypy3,py35,py36
+envlist = flake8,pypy3,py35,py36,py37,py38
 
 [flake8]
 ignore = F811,E226
-exclude = .tox/*,xx/*,__*,docs,*aio.py
```

Comment:
is it necessary diff in this commit?

Reply:
It allows to run CI on the asyncio related files so yes